src/cli/telemetry/telemetry.ts

```diff
@@ -141,21 +144,18 @@ export async function telemetryAction(action?: TelemetryCallback) {
   }
 }
 
+function hasAppTarget(): boolean {
```

Comment:
Can we get this under test please?

Reply:
I was under the impression that you only wanted exported members while we're getting started on coverage. Is that your goal?

Reply:
I'll being reorganizing the code to allow testing the object sent.